spec/system/dashboard/users_spec.rb

```diff
@@ -50,6 +50,9 @@
     end
 
     it 'can update user roles' do
+      location = FactoryBot.generate(:pumi_province)
+      allow(Pumi::Province).to receive(:all).and_return([location])
```

Comment:
Why do you need stub here?

Reply:
@samnang, i stub it because you told me that we shouldn't depend on third-party gems. Is there any other best way or idea?